pkg/skaffold/survey/survey.go

```diff
@@ -17,28 +17,38 @@ limitations under the License.
 package survey
 
 import (
+	"fmt"
 	"io"
 	"os"
-
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/color"
 )
 
 const (
 	Prompt = `Help improve Skaffold! Take a 10 seconds anonymous survey by running
    $skaffold survey`
+
+	Form = `Thank you for offering your feedback on Skaffold! Understanding your experiences and opinions helps us make Skaffold better for you and other users.
+   Our survey can be found here: https://forms.gle/BMTbGQXLWSdn7vEs6
+
+To permanently disable the survey prompt, run:
+   skaffold config set --survey --global disable-prompt true`
```

Comment:
This should also be in the `survey` help text.

Reply:
+1

Reply:
unfortunately, there is not method to add help in the `BuilderInterface`. This could be a big change since we might also need to make a change in `hack/man/`